backend/db/resolve_fetch_failures_0_members.py

```diff
@@ -70,17 +71,47 @@ def resolve_failures_0_members_if_exist(use_local_db=False, via_github_action=Tr
         else:
             resolve_fetch_failures_0_members(use_local_db=use_local_db)
 
+def filter_cset_id_where_0_expanded_members(cset_ids: List[int], schema=SCHEMA, use_local_db=False) -> List[int]:
+    """Filter cset IDs where there are 0 members when expanded
 
-def get_failures_0_members(version_id: Union[int, List[int]] = None, use_local_db=False) -> Tuple[Set[int], Dict[str, Dict]]:
+    TODO: complex cases: presently only does simple case: isExcluded=true, includeDescendants=false,
+     includeMapped=false. Ideally we'd also check cases where isExcluded=true, but if any includeDescendants=true,
+     check concept_ancestor to see if there actually are any descendants, and for includeMapped=true, check
+     concept_relationship to see if any mapped."""
+    qry = f"""SELECT codeset_id
+        FROM concept_set_version_item AS csvi
+        WHERE codeset_id {sql_in(cset_ids)}
+        GROUP BY codeset_id
+        HAVING COUNT(*) = COUNT(
+          CASE WHEN "isExcluded" = true AND "includeDescendants" = false AND "includeMapped" = false
+          THEN 1 ELSE NULL END);"""
+    with get_db_connection(schema=schema, local=use_local_db) as con:
+        id_rows: List[List[float]] = sql_query(con, qry, return_with_keys=False)
+        ids: List[int] = [int(x[0]) for x in id_rows]
+        return ids
+
+
+def get_failures_0_members(
+    version_id: Union[int, List[int]] = None, use_local_db=False
+) -> Tuple[Set[int], Dict[int, Dict]]:
     """Gets list of IDs of failed concetp set versions as well as a lookup for more information about them"""
-    version_id: List[int] = [version_id] if version_id and isinstance(version_id, int) else version_id
+    # Typing modification
+    version_id: List[int] = [version_id] if version_id and not isinstance(version_id, list) else version_id
 
+    # Lookup failure data in DB
     failures: List[Dict] = select_failed_fetches(use_local_db)
-    failure_lookup: Dict[str, Dict] = {x['primary_key']: x for x in failures}
-    failed_cset_ids: List[int] = version_id if version_id else [
-        int(x['primary_key']) for x in failures if x['status_initially'] == 'fail-0-members']
-    failed_cset_ids: Set[int] = set(failed_cset_ids)  # dedupe
-    return failed_cset_ids, failure_lookup
+    failure_lookup: Dict[int, Dict] = {int(x['primary_key']): x for x in failures}
+    failure_cset_ids: Set[int] = set(version_id if version_id else [
+        int(x['primary_key']) for x in failures if x['status_initially'] == 'fail-0-members'])
+
+    # Validate & filter
+    non_failures_passed: Set[int] = set(version_id).difference(failure_lookup.keys())
+    if bool(version_id and non_failures_passed):
+        print("Warning: Cset IDs were passed to be resolved, but these are no longer failures; skipping them: "
```

Comment:
## Additional small bugfix: `version_id` arg
It was possible to pass 1 or more `version_id` to `fetch_failures_0_members` to ask it to resolve the issue, but if one of the `version_id` had already resolved, this would trigger an error.